{{ cookiecutter.project_slug }}/.github/dependabot.yml

```
@@ -5,6 +5,16 @@
 
 version: 2
 updates:
+{%- if cookiecutter.packaging == 'uv' %}
+  - package-ecosystem: uv
```

This could be a one-liner to just change the package-ecosystem
```suggestion
  - package-ecosystem: {% if cookiecutter.packaging == "uv" %}uv{% else %}pip{% endif %}
```